src/main/config/util.ts

```diff
@@ -75,12 +76,17 @@ export function overwriteConfigData(
   parser.prop('useFiddler',          v => source.useFiddler          = !!v);
   parser.prop('disableExtremeGames', v => source.disableExtremeGames = !!v);
   parser.prop('showBrokenGames',     v => source.showBrokenGames     = !!v);
+  parser.prop('nativePlatforms',     v => source.nativePlatforms     = strArray(v));
   // Do some alterations
   source.flashpointPath = source.flashpointPath.replace(/\\/g, '/'); // (Clean path)
   // Return
   return source;
 }
 
+function strArray(strArray:any): string[] {
+  return (strArray || []);
```

Comment:
This will only replace falsely values with an empty array, and not ensure that the array or the items in it is of the correct type.

You should instead use ``Array.isArray(array)`` to see if the value is an array, and if it is you should map the values of it to a new array and coerce each element into a string. If it is not an array, then you could just return an empty array.

Reply:
In next commit